docs/aerodromes/Amberley.md

```diff
@@ -4,7 +4,78 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## YAMB Positions
 
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Amberley ADC    | Amberley Tower  | 118.300         | AM_TWR        |
+| Amberley SMC    | Amberley Ground  | 129.350      | AM_GND        |
+| Amberley ACD    | Amberley Delivery  | 134.600         | AM_DEL       |
+| Amberley ATIS    |   | 123.300         | YAMB_ATIS       |
+
+
+
+## Airspace
+
+AM ADC owns the Class C airspace in the AM CTR within 10nm of the YAMB ARP from `SFC` to `A015`. This airspace is primarily used for military circuits and initial and pitch approaches.
+
+
+## YAMB Aerodrome
+
+### SID Selection
+Aircraft planned via **BN**, **JEDDA**, **MESED**, **BOBOP**, **TATEN**, shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.
+Aircraft who are not planned via those points or who are negative RNAV may be assigned a RADAR sid or a visual departure. 
+
+#### Coded Departures
```

Comment:
I get the impression that these procedures are designed for high performance military jets, perhaps we could indicate that to avoid confusion with a civil aircraft wishing to perform a visual departure.